.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Changes must not contain commented code, unnecessary files, changes from other branches and generated files